docs/content/temporary-components.md

```diff
@@ -0,0 +1,43 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../README.md#recommended-use).
+
+-----
+
+
+## Temporary components
+
+The attribute `Improbable.Gdk.Core.RemoveAtEndOfTick`, can be applied to any ECS component, extending either `IComponentData` or `ISharedComponentData`.
```

Comment:
`Tick` > `Update` perhaps?

Reply:
It's removed at the end of the update look but it's not at the end of update. Also that is an implementation detail (although I'd be surprised if we made it so that SpatialOS ticks are not tied to update)
The intention was to show that it is based around the SpatialOS tick.

Reply:
We may need to link to what a `tick` is in that case, and what "being at the end of it" means specifically

Reply:
We should, but there isn't a doc for that yet. It will likely be in the system execution order doc.